src/api/public/apidocs-new/paths/search_request.yaml

```diff
@@ -0,0 +1,229 @@
+get:
+  summary: List request objects that match a XPath condition.
+  description: |
+    Return a collection of request objects that match a XPath condition.
+
+    This operation is the same as the one defined with [GET /search/request/id](#/Search/get_search_request_id) with the exception of the results returned.
+    While the former operation returns a list of request numbers, this one return a list of request objects. See example values of a succeeded request below.
```

Comment:
I would replace "former" by "previous" or "other".
"Former" is synonym of "ex", something that is not what it used to be anymore.

Reply:
It's true that _former_ can be a synonym of _ex_. However in this sentence, using `former` is fine as @eduardoj is referring to the endpoint he previously mentioned, before mentioning a second endpoint with _this one_.

_Former_ is often use to denote the first or first mentioned of two people or things.

Reply:
> I would replace "former" by "previous" or "other". "Former" is synonym of "ex", something that is not what it used to be anymore.

"Former" can be used as "first in order of two or more things cited or understood". See [this reference](https://www.merriam-webster.com/dictionary/former). I would leave the sentence as it is, unless you have a strong opinion on it.

Reply:
Understood, thanks for the explanations